js/scripts/prop-types.js

```diff
@@ -22,29 +22,31 @@ _.extend(BaseType.prototype, {
     },
 })
 
-function ThreeType(typeName) {
+function ThreeType(typeName, nullable=true) {
```

Comment:
maybe, for future-proofing, might be better to expose this as an `options` param, so the method header won't change as often, e.g.:
```
function ThreeType(typeName, options={}) {
  options = loadDefaults(options, ThreeType.DEFAULTS); // pseudocode for getting default option vals
  if (options.nullable) { ... }
}
```